style-dictionary/classic/index.ts

```diff
@@ -9,52 +9,48 @@ import {
   createFlashbarContext,
   createFlashbarWarningContext,
   createTopNavigationContext,
-} from '../utils/contexts';
-import { createColorMode, createDensityMode, createMotionMode } from '../utils/modes';
+} from '../utils/contexts.js';
+import { StyleDictionary } from '../utils/interfaces.js';
+import { createColorMode, createDensityMode, createMotionMode } from '../utils/modes.js';
 
 const modes = [
   createColorMode('.awsui-dark-mode'),
   createDensityMode('.awsui-compact-mode'),
   createMotionMode('.awsui-motion-disabled'),
 ];
 
-const tokenCategories = [
-  require('./color-palette'),
-  require('./color-charts'),
-  require('./color-severity'),
-  require('./colors'),
-  require('./typography'),
-  require('./borders'),
-  require('./motion'),
-  require('./sizes'),
-  require('./spacing'),
-  require('./shadows'),
+const tokenCategories: Array<StyleDictionary.CategoryModule> = [
```

Comment:
Return value of `require()` calls was any. Switching to `import()` enabled actual type validation, which required some fixes